### PR TITLE
[NC-591] Add autofocus option to TextBox and TextArea native widgets

### PR DESCRIPTION
**Pull request by Ahmad-Elsayed**
Add docs for autofocus feature in text area and text box widgets

To be released with Studio Pro 9.15

### CODE REVIEW

File: content/en/docs/refguide/modeling/pages/input-widgets/text-area.md [hunk 1]
```diff
@@ -129,6 +129,11 @@ The autocomplete property specifies if a text area should have autocomplete enab
 {{% alert color="info" %}}This option is only available in native pages.{{% /alert %}}
 {{% alert color="info" %}}In Android when autocomplete is turned off it will remove support for new lines.{{% /alert %}}
 
+#### 2.8.5 AutoFocus
+
+If `true`,  the TextArea will be rendered focused and the device keyboard is opened.
```
Comment (pratikgoyal9) [inline content/en/docs/refguide/modeling/pages/input-widgets/text-area.md]: It should be "text area"

File: content/en/docs/refguide/modeling/pages/input-widgets/text-box.md [hunk 2]
```diff
@@ -176,6 +176,11 @@ The placeholder text is shown when no text has been entered yet, or when a displ
 
 It can be used, for example, to give a hint to the end-user what kind of text should be entered.
 
+#### 2.8.5 AutoFocus
+
+If `true`,  the Textbox will be rendered focused and the device keyboard is opened.
```
Comment (pratikgoyal9) [inline content/en/docs/refguide/modeling/pages/input-widgets/text-box.md]: It should be "text box"

**Review by pratikgoyal9 — CHANGES_REQUESTED** [review]

File: content/en/docs/refguide/modeling/pages/input-widgets/text-area.md (hunk 1 above)
Comment (Ahmad-Elsayed) [reply content/en/docs/refguide/modeling/pages/input-widgets/text-area.md]: done

File: content/en/docs/refguide/modeling/pages/input-widgets/text-box.md (hunk 2 above)
Comment (Ahmad-Elsayed) [reply content/en/docs/refguide/modeling/pages/input-widgets/text-box.md]: done

File: content/en/docs/refguide/modeling/pages/input-widgets/text-area.md (hunk 1 above)
Comment (Ahmad-Elsayed) [reply content/en/docs/refguide/modeling/pages/input-widgets/text-area.md]: V 9.15

**Review by pratikgoyal9 — APPROVED** [review]

**Review by ConnorLand — APPROVED** [review]

**Review by ConnorLand — APPROVED** [review]

**Pull request merged**